airqo_monitor/models.py

```diff
@@ -1,5 +1,67 @@
 from django.db import models
 
-# Create your models here.
+
 class Greeting(models.Model):
     when = models.DateTimeField("date created", auto_now_add=True)
+
+
+class Incident(models.Model):
+
+    class Meta:
+        db_table = 'incident'
+
+    created_at = models.DateTimeField("date created", auto_now_add=True)
+    channel_id = models.IntegerField(null=False, db_index=True)
+
```

Comment:
maybe a fixed_at or something?

Reply:
yes we should add but i want to add it in a later PR once we've decided on what that flow will look like